docs/tsg.md

```diff
@@ -151,6 +152,16 @@ For **Standard** instances, **concurrent** connection count limit **per unit** i
 
 The connections include both client and server connections. check [here](https://docs.microsoft.com/en-us/azure/azure-signalr/signalr-concept-messages-and-connections#how-connections-are-counted) for how connections are counted.
 
+<a name="429_negotiate"></a>
+## 429(Too Many Requests) returned for negotiate requests
```

Comment:
merge with the above, negotiate 429 is one condition for client 429. Also describe the message client would see if negotiate 429